examples/README.md

```diff
@@ -0,0 +1,11 @@
+# How to get started with Windows 10
+
+Easiest might be to start with the sshd image. On Windows 10 Professional or Windows 10 Enterprise you can build the image as follows:
+
+`linuxkit.exe build -format iso-efi sshd.yml`
+
+Then you run the image, maybe in Git BASH or some other shell which respects the `$HOME/.ssh/id_rsa.pub` variable:
```

Comment:
I think it is the `linuxkit build` command which needs to find `$HOME/.ssh/id_rsa.pub`, not the `linuxkit run`